lib/cmr/cmr_client.rb

```diff
@@ -42,6 +42,16 @@ def get_collections_by_post(query, token = nil)
       post(url, query.to_query, headers.merge(token_header(token)))
     end
 
+    def get_variables(options = {}, token = nil)
```

Comment:
If this is the same as the method below (`search_collections`) we should keep the naming consistent.